README.md

```diff
@@ -18,6 +18,8 @@ This repository contains Drash-approved middleware modules that you can use in y
 
 * [Dexter](./dexter) - Dexter is a logging middleware inspired by [expressjs/morgan](https://github.com/expressjs/morgan).
 
+* [Paladin](./paladin) - Paladin helps secure your Drash applications through headers. Inspired by [helmet](https://github.com/helmetjs/helmet)
```

Comment:
Paladin helps you secure your Drash applications by setting various HTTP headers.

Reply:
https://github.com/drashland/deno-drash-middleware/pull/25/commits/b8b3bba86166d57696b710bdb3513861ded928cd